samples/chat-app/server/Schema.fs

```diff
@@ -0,0 +1,759 @@
+namespace FSharp.Data.GraphQL.Samples.ChatApp
+
+open FSharp.Data.GraphQL
+open FSharp.Data.GraphQL.Types
+open System
+
+type Root =
+  { RequestId : string }
+
+type FakePersistence() =
```

Comment:
I would suggest moving this class to a separate file.

Reply:
Done.